LinuxPBA/UnlockSEDs.cpp

```diff
@@ -35,7 +35,11 @@ uint8_t UnlockSEDs(char * password) {
     LOG(D4) << "Enter UnlockSEDs";
     mvprintw(6,2,"Scanning....");
     while (TRUE) {
-        snprintf(devref,23,"/dev/sd%c",(char) 0x61+i);
+        switch(DtaDevOS::getNextDevice(i)){
+            case 1  : snprintf(devref,23,"/dev/%s",DtaDevOS::getDeviceName()); break;
+            case -1 : snprintf(devref,23,"/dev/sd%c",(char) 0x61+i);    break;
```

Comment:
@laser-beans are you able to change this timely, or should we do this later after merging this code?